### ISSUE TITLE
Feature Request: Battery SOC

### ISSUE THREAD

**Issue by fwPunsher** [deg0nz/MMM-EnergyMonitor#11]
Hi,

would be a nice addition to display the state of charge of the battery.
So potentially add a label with the percentage next to the current symbol. And as a bonus switch the icon between a few variations (full, medium, empty etc.).

Best Regards

**Comment by deg0nz** [comment]
Yeah, you are probably right. I didn't implement it, because the companion module [MMM-VartaESS](https://github.com/deg0nz/MMM-VartaESS) already shows the battery state on my mirror.

It's definitely useful. We can implement that. I don't know, if I will have time in the next weeks. PRs are very welcome ;)
